doc/index.rst

```diff
@@ -0,0 +1,22 @@
+.. Example Application documentation master file, created by
+   sphinx-quickstart on Wed Apr  3 15:28:39 2024.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Example Application's documentation!
+===============================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+This is a simple Sphinx documentation setup for ``example-application``. You can
+reference Zephyr documentation items, like
+:external+zephyr:ref:`getting_started` or :external+zephyr:c:func:`k_sleep`.
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
```

Comment:
this does not seem to link correctly from the artifact generated page, points to `http://localhost:8000/sphinx/py-modindex.html` which is not there

```console
~/x$ ls sphinx/
_sources/  genindex.html  objects.inv  searchindex.js
_static/   index.html     search.html
```

Reply:
removed modindex entry, there are no Python modules to be listed